client/src/core/SurveyPopup.js

```diff
@@ -161,18 +176,29 @@ export const SurveyPopup = withRouter(
               {text_maker("suvey_popup_header")}
             </Fragment>
           }
-          body={<TM k="survey_popup_body" />}
+          body={
+            <div
+              style={{
+                fontSize: "0.9em",
+                display: "flex",
+                flexDirection: "column",
+              }}
+            >
+              <TM k="survey_popup_body" />
+              {full_survey_button}
+              <TM
+                k="take_simplified_survey_text"
+                style={{ margin: "8px 0px" }}
+              />
+              <EmailFrontend
+                template_name="feedback_simplified"
```

Comment:
Reminder to rebuild email_backend before merging! (Also reminder that test only works local for now until then)